sklearn/datasets/rcv1.py

```diff
@@ -0,0 +1,201 @@
+"""RCV1 dataset.
+"""
+
+# Author: Tom Dupre la Tour
+# License: BSD 3 clause
+
+import sys
+import logging
+
+from os.path import exists, join
+from gzip import GzipFile
+from io import BytesIO
+try:
+    from urllib2 import urlopen
+except ImportError:
+    from urllib.request import urlopen
+
+import numpy as np
+import scipy.sparse as sp
+
+from .base import get_data_home
+from .base import Bunch
+from .base import _mkdirp
+from ..externals import joblib
+from .svmlight_format import load_svmlight_files
+from ..utils import shuffle as shuffle_
+
+
+URL = ('http://jmlr.csail.mit.edu/papers/volume5/lewis04a/'
+       'a13-vector-files/lyrl2004_vectors')
+URL_topics = ('http://jmlr.csail.mit.edu/papers/volume5/lewis04a/'
+              'a08-topic-qrels/rcv1-v2.topics.qrels.gz')
+
+logger = logging.getLogger()
+
+
+def fetch_rcv1(data_home=None, download_if_missing=True,
+               random_state=None, shuffle=False):
+    """Load the RCV1 dataset, downloading it if necessary.
```

Comment:
Can you mention which version of rcv1 is loaded (rcv1.binary, rcv1.multiclass, rcv1v2(topics; subsets), rcv1v2(topics; full sets), rcv1v2(industries; full sets), ...)?